template/lambda-sqs-worker/src/app.ts

```diff
@@ -36,11 +36,17 @@ export const handler = createHandler<SQSEvent>(async (event) => {
 
   const record = event.Records[0];
 
+  const body = record?.body;
+
+  if (!body) {
+    throw new Error(`Record: ${JSON.stringify(record)} is malformed`);
+  }
```

Comment:
I'm a bit cautious of logging the whole `record`. Could we do something like:

```suggestion
  if (!record) {{
    throw new Error('Malformed SQS event with no records');
  }
  
  // Or just refer to `record.body` below
  const { body } = record;
```